tools/api/src/main/java/org/wildfly/transformer/tool/api/ToolUtils.java

```diff
@@ -70,6 +70,7 @@ public static void transformClassFile(final File inClassFile, final File targetD
     }
 
     public static void transformJarFile(final File inJarFile, final File targetDir, final String packagesMappingFile) throws IOException {
+        if (!targetDir.exists()) throw new IOException("Couldn't find directory: " + targetDir.getAbsolutePath());
```

Comment:
Change looks good.  Should we also check if targetDir is the folder that contains inJarFile and throw an exception or should we allow that?

Perhaps something like checking if the following is true ``` boolean outputToInputFile = targetDir.toPath().equals(inJarFile.toPath().toRealPath().getParent())```

Reply:
We can discuss my feedback on email later, no need to block this change.